scss/bitstyles/settings/_typography-responsive.scss

```diff
@@ -1,23 +1,66 @@
 @use './setup';
 @use '../tools/font-size';
+@use '../tools/line-height';
 
-$font-sizes: (
+$typographic-scale: (
   '#{setup.$no-media-query}': (
-    'h1': font-size.get('6'),
-    'h2': font-size.get('5'),
-    'h3': font-size.get('4'),
-    'h4': font-size.get('3'),
-    'h5': font-size.get('2'),
-    'h6': font-size.get('1'),
-    'body': font-size.get('2'),
+    'h1': (
+      'font-size': font-size.get('6'),
+      'line-height': line-height.get('1'),
+    ),
+    'h2': (
+      'font-size': font-size.get('5'),
+      'line-height': line-height.get('2'),
+    ),
+    'h3': (
+      'font-size': font-size.get('4'),
+      'line-height': line-height.get('3'),
+    ),
+    'h4': (
+      'font-size': font-size.get('3'),
+      'line-height': line-height.get('4'),
+    ),
+    'h5': (
+      'font-size': font-size.get('2'),
+      'line-height': line-height.get('5'),
+    ),
+    'h6': (
+      'font-size': font-size.get('1'),
+      'line-height': line-height.get('6'),
+    ),
+    'body': (
+      'font-size': font-size.get('2'),
+      'line-height': line-height.get('6'),
+    ),
   ),
   'm': (
-    'h1': font-size.get('8'),
-    'h2': font-size.get('7'),
-    'h3': font-size.get('6'),
-    'h4': font-size.get('4'),
-    'h5': font-size.get('2'),
-    'h6': font-size.get('1'),
-    'body': font-size.get('2'),
+    'h1': (
+      'font-size': font-size.get('8'),
+      'line-height': line-height.get('1'),
+    ),
+    'h2': (
+      'font-size': font-size.get('7'),
+      'line-height': line-height.get('2'),
+    ),
+    'h3': (
+      'font-size': font-size.get('6'),
+      'line-height': line-height.get('3'),
+    ),
+    'h4': (
+      'font-size': font-size.get('4'),
+      'line-height': line-height.get('4'),
+    ),
+    'h5': (
+      'font-size': font-size.get('2'),
+      'line-height': line-height.get('5'),
+    ),
+    'h6': (
+      'font-size': font-size.get('1'),
+      'line-height': line-height.get('6'),
+    ),
+    'body': (
+      'font-size': font-size.get('2'),
+      'line-height': line-height.get('6'),
```

Comment:
This seems a bit large as a base line-height — something around 1.4 is probably more comfortable to read